frontend/src/components/permit/PermitCard.vue

```diff
@@ -158,15 +165,25 @@ p {
 }
 
 .p-card {
-  border-style: solid;
-  border-width: 1px;
-
+  border-radius: 16px;
   :deep(.p-card-content) {
     padding-bottom: 0;
   }
+
+  :deep(.p-card-body) {
+    border-style: solid;
+    border-width: 1px;
+    border-radius: 16px;
+    border-color: #05366260;
+    box-shadow: 0px 4px 4px 0px #0000001a;
+  }
```

Comment:
UX: Double check the guidance w.r.t. sizing. IIRC the industry prefers to lean towards relativistic measurement units like em or rem in lieu of px, especially when considering high DPI monitors.